test/test-helpers/test-env.js

```diff
@@ -1,3 +1,6 @@
 require('module-alias/register')
 const path = require('path')
 require('dotenv').config({ path: path.join(__dirname, '../test.env') })
+const sinonChai = require('sinon-chai')
+const chai = require('chai')
+chai.use(sinonChai)
```

Comment:
Do these need to be here?

Reply:
They enable the use of sinonChai matchers without having to require the dependency in every test